tests/org.eclipse.ui.tests/META-INF/MANIFEST.MF

```diff
@@ -40,7 +40,11 @@ Require-Bundle: org.eclipse.core.resources;bundle-version="3.14.0",
  org.eclipse.e4.ui.workbench.renderers.swt;bundle-version="0.10.0",
  org.mockito.mockito-core;bundle-version="2.13.0",
  org.eclipse.ui.views.log;bundle-version="1.2.1300",
- org.eclipse.jdt.ui
+ org.eclipse.jdt.ui,
```

Comment:
AFAIK we can't use any jdt test bundles here

Reply:
Raghu, all the jdt ui related tests should go together with the jdt ui patch, not with this one.

Reply:
removed jdt ui tests dependecy. converted java project normal project.